### PR TITLE
Fixed main menu + report/coach buttons

### PR DESCRIPTION
**Pull request by BingBongBonky**
Changelog here 
https://github.com/BingBongBonky/SpaceHUD/releases/tag/1.3.0

### CODE REVIEW

**Review by rbjaxter — APPROVED** [review]

File: hud-data/spacehud.json [hunk 1]
```diff
@@ -37,10 +37,11 @@
       "tc"
     ],
     "res": ["4/3", "5/4", "16/9", "16/10"]
+    "lang": ["en"]
```
Comment (rbjaxter) [inline hud-data/spacehud.json:40]:
```suggestion
    "res": ["4/3", "5/4", "16/9", "16/10"],
    "lang": ["en"]
```

**Review by rbjaxter — CHANGES_REQUESTED** [review]
The check appears to be mad about a few things I'm not sure about, though I'm guessing this is one of them

**Review by rbjaxter — DISMISSED** [review]
looks good to me 👍

**Comment by BingBongBonky** [comment]
Alright we're good now I forgot to add the scoreboard fix

**Review by rbjaxter — APPROVED** [review]

**Pull request merged**